build.proj

```diff
@@ -509,11 +509,6 @@
     <Message Importance="high" Text="Running Dependency Analysis..." />
     <Exec Command="$(MSBuildProjectDirectory)\src\Package\StaticAnalysis.exe $(MSBuildProjectDirectory)\src\Package\$(Configuration) $(MSBuildProjectDirectory)\src\Package true $(SkipHelp)" Condition="'$(Latest)' == 'true'"/>
     <Exec Command="$(MSBuildProjectDirectory)\src\Package\StaticAnalysis.exe $(MSBuildProjectDirectory)\src\Stack\$(Configuration) $(MSBuildProjectDirectory)\src\Stack" Condition="'$(Stack)' == 'true'" ContinueOnError="True"/>
-    <OnError ExecuteTargets="StaticAnalysisErrorMessage"/>
-  </Target>
-
-  <Target Name="StaticAnalysisErrorMessage">
```

Comment:
Why are you reverting this change?

Reply:
@maddieclayton looks like it a result of a weird merge